app/models/video.py

```diff
@@ -0,0 +1,22 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+from sqlalchemy.orm import relationship 
+
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer, default=0) 
+    available_inventory = db.Column(db.Integer, default=total_inventory) 
+    # rentals = db.relationship('Rental', back_populates='video', lazy=True)
```

Comment:
Yes! This comment is one way of setting up the one-to-many relationship with videos on rental